TheengsGateway/ble_gateway.py

```diff
@@ -22,6 +22,7 @@
 # python 3.6
 
 import asyncio
+from distutils.command.config import config
```

Comment:
I'm not sure why we need this?

Reply:
Same, I´m not sure why VSCode added it without me noticing it.